library/src/main/java/com/cube/storm/ui/lib/adapter/StormSpotlightAdapter.java

```diff
@@ -95,10 +98,31 @@ public Object instantiateItem(@NonNull ViewGroup container, int position)
 		final SpotlightImageProperty spotlightItem = getItem(position);
 		if (spotlightItem != null)
 		{
+			View textContainer = view.findViewById(R.id.text_container);
 			ImageView imageView = view.findViewById(R.id.image_view);
-			TextView textView = view.findViewById(R.id.text_ticker);
+			TextView category = view.findViewById(R.id.category);
+			TextView title = view.findViewById(R.id.title);
+			TextView description = view.findViewById(R.id.description);
+
 			imageView.populate(spotlightItem.getImage());
-			textView.populate(spotlightItem.getText(), spotlightItem.getLink());
+			category.populate(spotlightItem.getCategory());
+			title.populate(spotlightItem.getTitle());
```

Comment:
Could you make this field fallback to `getText()` if `getTitle()` is missing? This will ensure the new-style Spotlight is backwards compatible with the old `SpotlightItem` version (the one that uses text)

Reply:
Good idea, I've just done this is the `public class SpotlightImageProperty`, how the other classes do it.